extensions/visualizations/visualizations.js

```diff
@@ -74,3 +74,22 @@ oppia.directive('oppiaVisualizationFrequencyTable', [function() {
     ]
   };
 }]);
+
+oppia.directive('oppiaVisualizationEnumeratedFrequencyTable', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    templateUrl: 'visualizations/EnumeratedFrequencyTable',
+    controller: [
+      '$scope', '$attrs', 'HtmlEscaperService',
+      function($scope, $attrs, HtmlEscaperService) {
+        $scope.data = HtmlEscaperService.escapedJsonToObj($attrs.data);
+        $scope.options = HtmlEscaperService.escapedJsonToObj($attrs.options);
+        $scope.answerVisible = new Array($scope.data.length).fill(false);
```

Comment:
.fill() isn't supported in IE, maybe use something more compatible? (It's OK if the code ends up a bit longer.)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill

Reply:
Done.